src/emulator/host/include/host/state.h

```diff
@@ -29,7 +30,6 @@
 #include <nids/types.h>
 #include <np/state.h>
 #include <renderer/state.h>
-#include <gdbstub/state.h>
```

Comment:
clang-format also did this in an attempt to make it all nice and alphabetical, but it ended up breaking the windows build (state.h needs windows.h to be included before) it. I'll address all of these things. Thanks for letting me know :D